ballerina/Ballerina.toml

```diff
@@ -5,8 +5,8 @@ name = "zoom.meetings"
 version = "1.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = [] # TODO: Add keywords
+# icon = "icon.png" # TODO: Add icon
```

Comment:
Add keywords and icons. Possible keywords: `["zoom", "video conference", "meetings"]`